proto/uber/cadence/api/v1/service_domain.proto

```diff
@@ -78,7 +78,7 @@ message RegisterDomainRequest {
   ArchivalStatus visibility_archival_status = 12;
   string visibility_archival_uri = 13;
   map<string, string> active_clusters_by_region = 14; // todo (david.porter) to remove as this is no longer used
-  ActiveClusters active_clusters_by_cluster_attribute = 15;
```

Comment:
This is a breaking change, but this feature's not implemented yet, so it shouldn't be a concern. 

The actual field has - inside it - fields by cluster-attributes, so adding it externally here as a name is redundant